flink-connector-postgres-cdc/src/main/java/io/debezium/connector/postgresql/PostgresStreamingChangeEventSource.java

```diff
@@ -113,12 +112,17 @@ public PostgresStreamingChangeEventSource(
 
     @Override
     public void init() {
+        // It's not necessary to refresh schema again, which is very time-consuming.
+        // The schema of taskContext is the reference of PostgresSourceFetchTaskContext#schema, and
+        // has been initialized when submit StreamSplit fetch task by
+        // IncrementalSourceStreamFetcher#submitTask -> PostgresSourceFetchTaskContext#configure.
+
         // refresh the schema so we have a latest view of the DB tables
-        try {
-            taskContext.refreshSchema(connection, true);
```

Comment:
It's copied form debezium.  Maybe we can provide a subclass(StreamSplitReadTask) to override this method?